src/Network/Network/VirtualRouter/NewAzureRMVirtualRouterCommand.cs

```diff
@@ -49,18 +49,30 @@ public partial class NewAzureRmVirtualRouter : VirtualRouterBaseCmdlet
 
         [Parameter(
             Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
```

Comment:
this is breaking change since default parameter set was changed and a mandatory parameter has changed input type